app/views/root/place.html.erb

```diff
@@ -10,17 +10,21 @@
 
         <%= raw @publication.introduction %>
 
-        <%= render :partial => 'location_form', :locals => {:format => 'service'} %>
+        <%= render partial: 'location_form', locals: { format: 'service' } %>
       </div>
     </div>
   </section>
 
   <% if @postcode.present? and !@publication.places.nil? %>
     <% if @publication.places.any? %>
-      <section class="places">
-        <h2>Results near <strong><%= @postcode %></strong>:</h2>
+      <section class="places"
+               data-module="auto-track-event"
+               data-track-category="placeSearch"
+               data-track-action="postcodeResultShown"
+               data-track-label="<%= places_names(@publication.places) %>">
```

Comment:
As discussed face-to-face: I'm mildly wary of this.  For a standard "find-my-nearest" we would be sending the names of 10 things which could be fairly large, and names are the sort of thing that might change over time.  We also have the issue that we join the names on `,`, but `,` can be present within the name, so performing an unjoin operation would be hard.  We could use the `CSV` library to get around this - but maybe other punctuation could be considered (wrapping in `[]` and joining on `,` might work).